src/main/java/frc/robot/common/hardware/ColorSensorMuxed.java

```diff
@@ -28,6 +30,7 @@ public ColorSensorMuxed(int... ports) {
         i2cPorts.add(p);
         // Initialize each device, only need to keep last object
         sensors = new ColorSensorV3(Port.kOnboard);
+        Shuffleboard.addEventMarker("Sensors Activated", EventImportance.kCritical);
```

Comment:
Not sure that this is a good event.  It happens at the beginning of the match for every match.  And only once per match

This is the initialization of the sensors.

Reply:
Changed it to say sensors initialized, and added a new Shuffleboard event for when they are on and starting to sense color.